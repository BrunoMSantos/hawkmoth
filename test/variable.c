/**
 * This is a variable document.
 */
static int sheesh;
