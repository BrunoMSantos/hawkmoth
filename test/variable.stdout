
.. c:var:: int sheesh

   This is a variable document.

